packages/blade/src/components/Tooltip/_decisions/decisions.md

```diff
@@ -0,0 +1,221 @@
+# Tooltip API Decisions <!-- omit in toc -->
+
+The tooltip typically provides additional context about the element or its function. A tooltip is always triggered by a `mouse hover` on desktop and `on tap` on mobile.
+
+<img src="./tooltip-thumbnail.png" width="380" alt="" />
+
+- [Design](#design)
+- [`Tooltip` API](#tooltip-api)
+  - [Examples:](#examples)
+    - [Usage](#usage)
+  - [Working with non-interactive triggers](#working-with-non-interactive-triggers)
+  - [Custom Triggers](#custom-triggers)
+- [Library](#library)
+  - [Bundle Size](#bundle-size)
+- [Motion](#motion)
+- [Accessibility](#accessibility)
+- [Open Questions](#open-questions)
+- [References](#references)
+
+## Design
+
+[Figma Link](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?type=design&node-id=40540-559304&t=tmTrf3xJU6oj59fM-0) to all variants of the Tooltip component
+
+## `Tooltip` API
+
+| Prop         | Type                                                                     | Default     | Description                                                                                                                             | Required |
+| ------------ | ------------------------------------------------------------------------ | ----------- | --------------------------------------------------------------------------------------------------------------------------------------- | -------- |
+| content      | `string`                                                                 | `undefined` | Content of the tooltip                                                                                                                  | ✅       |
```

Comment:
We have a use-case for having a link inside Tooltip. Please check the coin at the top of this page: https://razorpay.com/payment-pages/

Reply:
@cseas Tooltip should only contain non-interactive elements inside of it for accessibility reasons. 

If you want to add interactive element inside like a button or link, a Popover or Toggletip will be a valid pattern where users will click the trigger and the Popover won't dismiss automatically or just by hovering out of the trigger. 

<img width="802" alt="image" src="https://github.com/razorpay/blade/assets/35374649/0e868822-c89a-4ed7-954b-ef71afd2cb45">

https://web.dev/building-a-tooltip-component/#accessibility

Reply:
Yes, we will release Popover as a separate component.